### ISSUE TITLE
missing docs for filtered joins?

### ISSUE THREAD

**Issue by trondhindenes** [piccolo-orm/piccolo#546]
I'm basically trying to do:
`await Band.objects(Band.manager).where(Band.manager.name == "peter")`

is that not supported? I find very little docs about filtered joins (is that the correct term?) in the docs

**Comment by dantownsend** [comment]
Yeah, that query should work. Just took a look at the docs, and you're right - it doesn't seem to say anything.

Should probably say something on this page:

https://piccolo-orm.readthedocs.io/en/latest/piccolo/query_clauses/where.html

**Comment by trondhindenes** [comment]
thanks - will keep troubleshooting then, just wanted to check that I'm not doing something unsupported. Thanks (as always) for the quick response!
